shared/validate/schema/model/transaction/getTransaction.ts

```diff
@@ -60,4 +60,24 @@ export const SCHEMAS_TRANSACTIONS = [
         },
         required: ['blockId', 'limit', 'offset']
     },
+    {
+        id: API_ACTION_TYPES.GET_TRANSACTIONS_BY_HEIGHT,
```

Comment:
maybe from height? Because here is limit+offset

Reply:
no, from heigh